src/pydvl/influence/general.py

```diff
@@ -68,15 +70,14 @@ def calculate_influence_factors(
     :param inversion_func: function to use to invert the product of hvp (hessian vector product) and the gradient
         of the loss (s_test in the paper).
     :param lam: regularization of the hessian
-    :param progress: True for plotting the progress bar, False otherwise.
-    :returns: A np.ndarray of size (N, D) containing the influence factors for each dimension (D) and test sample (N).
     :param progress: If True, display progress bars.
+    :returns: A np.ndarray of size (N, D) containing the influence factors for each dimension (D) and test sample (N).
     """
     if not _TORCH_INSTALLED:
         raise RuntimeWarning("This function requires PyTorch.")
     grad_xy, _ = model.grad(x, y)
-    hvp = lambda v: model.mvp(grad_xy, v, progress=progress) + lam * v
-    test_grads = model.split_grad(x_test, y_test, progress=progress)
+    hvp = lambda v: model.mvp(grad_xy, v) + lam * v
```

Comment:
You don't pass the progress parameter to the `mvp` method anymore?

Reply:
No because in conjugate gradient hvp is called several times with only one element (and so progress bar gets spammed).